### ISSUE TITLE
Editing line comments in the programming-exercise assessment editor does not work

### ISSUE THREAD

**Issue by moritzhertler** [ls1intum/Artemis#3454]
### Describe the bug
Sometimes the edit buttons of all line comments in one file do not work. All the comments in other files are unaffected. Saving the work and reloading resolves the issue. This happens annoyingly often (I had to reload the site up to five times while grading one submission).

#### To Reproduce

The issue appears randomly in different submissions. I could not find a pattern. 

### Environment

<details><pre>
 - OS: Windows 10 20H2
 - Browser: Chrome
 - Version: 90.0.4430.212 
</pre></details>

#### Additional context
I am a Tutor at KIT, @dfuchss is my supervisor.

#### Output in Web Console

<details><pre>
  The console does not output anything when pressing the non-functioning edit button.
</pre></details>

**Comment by krusche** [comment]
Can you provide some screenshots (if needed black out the details that should not be public) or even a screencast?
Did you find any pattern when you encountered the issue?

Should I invite you to our Slack workspace? It might be easier to talk directly to a developer who could fix the issue, then you could also look together over the issues

**Comment by moritzhertler** [comment]
I started a new assessment and added/edited random comments until the issue appeared again.

**The screencast can be viewed [here](https://drive.google.com/file/d/1FzZeMp-c4FQQBTBVjMS5bZ26B_xY-7Eu/view?usp=sharing).**

Short description of the screencast:

- I had already added a few comments before this recording. The cast starts with me editing existing comments that have already been saved.
- At 00:21 the editing button of the comment on line 11 in Months.java does not work anymore. The console does not show any output.
- I continue to try to edit different comments in different files. 
- At 01:08 a second comment on line 12 in Weekdays.java is affected. 
- I continue to try to edit different comments in different files. 
- After reloading at 02:05 the issues are resolved.

It is probably a coincidence but all affected comments so far have had a score of 0 points. This could be because I generally use comments with a 0 points score way more than comments with a negative or positive score.

I do not know if this could be relevant but I usually have the assessment dashboard opened in a second tab to get an overview of how many exercises I have already graded. As far as I remember, the issue only appeared when the second Artemis tab is opened (which is most of the time so likely also a coincidence).

I could not find any other patterns.

Thank you for the offer to join the Slack workspace. I would like to look over the issue with a developer.
